### ISSUE TITLE
Desktop mode no longer displays the desktop versions of websites

### ISSUE THREAD

**Issue by heres-maria** [MozillaReality/FirefoxReality#2885]
**[Affected Versions]:**
- Firefox Reality 9 RC 4 (c64365e3)

**[Affected Platforms]:**
- HTC Vive Focus Plus - Android 7.1.2
- HTC Vive Focus - Android 7.1.2
- Oculus Quest - Android 7.1.1
- Oculus Go - Android 7.1.2
- Pico Neo 2 - Android - 8.1.0

**[Prerequisites]:**
- The headset is turned on, the browser is installed and open.
- The “User-Agent Mode” is set to “Desktop”.

**[Steps to reproduce]:**                                        
1. Go to [Twitch](https://www.twitch.tv/).
2. Observe the URL of the website as well as the website itself.

**[Expected results]:**
- The URL is the desktop version one and the desktop version of the website is displayed.

**[Actual results]:**
- The URL is the mobile version one and the mobile version of the website is displayed.

**[Notes]:**
- This issue is reproducible for other websites as well.
- Attached is a [recording](https://drive.google.com/file/d/1WoJkeJkyO_Nqcpin9w6621p5O4GTtoU5/view?usp=sharing) of the issue.

**Comment by MortimerGoro** [comment]
It have tries this and it seems that twitch only checks the URL to decide between mobile/desktop viewports
- https://m.twitch.tv/ Shows mobile viewport no matter of the device/desktop mode (Tried in FxR, Oculus Broser, FxR desktop)
- https://www.twitch.tv/ Shows desktop viewport no matter of the device/desktop mode (Tried in FxR, Oculus Broser, FxR desktop)

**Comment by MortimerGoro** [comment]
I tried on FxR v8 and it works as expected there. I'm going to research what happened

**Comment by bluemarvin** [comment]
UA override is broken in GV when e10s is turned off: https://bugzilla.mozilla.org/show_bug.cgi?id=1618649

**Comment by heres-maria** [comment]
I have verified this issue and it is no longer reproducible with the latest Firefox Reality v9 RC5 (3e30ba6e)  build.
The "Desktop" user agent mode has been removed from the "Display" menu settings. Using the hamburger menu button correctly changes the user agent.
Tested on Pico Neo 2 (Android 8.1.0), Oculus Go (Android 7.1.2), Oculus Quest (Android 7.1.1), HTC Vive Focus Plus (Android 7.1.2), HTC Vive Focus (Android 7.1.2) and Pico Neo 2 (Android 8.1.0).
